app/views/settings/ext_cert/index.html.slim

```diff
@@ -0,0 +1,138 @@
+= render 'settings/apply'
+
+h2 External Certificates
+
+p Enable the use of External SSL Certificates for CaaSP and Kubernetes services.  Certificates and matching keys must be in armored PEM format.
+.alert.alert-warning role='alert'
+ p Remember to add a Root CA Certificate to the System Wide Certificates corresponding to any new External Certificates
+
+= form_for :external_certificate, url: settings_ext_cert_index_path, method: :post do |f|
+  br
+  .panel.panel-default
```

Comment:
Extract all the panels to its own partial.

Reply:
I presume you're looking for something like this:
```
  .panel
    .panel-default
      .panel-heading
        h3.panel-title Dex SSL Certificate and Key
```

Reply:
No. It's about extracting the content to a new file and use `= render "settings/ext_cert/_partial_name"`. This way the templates become more readable. 

Remember to pass the form reference as a parameter.

Reply:
Done